src/Floskell/ConfigFile.hs

```diff
@@ -27,12 +28,19 @@ import           Control.Applicative        ( (<|>) )
 import           Data.Aeson
                  ( (.:?), (.=), FromJSON(..), ToJSON(..) )
 import qualified Data.Aeson                 as JSON
-import qualified Data.Aeson.Parser          as JSON ( json' )
 import qualified Data.Aeson.Types           as JSON ( typeMismatch )
 import qualified Data.Attoparsec.ByteString as AP
 import qualified Data.ByteString            as BS
 import           Data.Char                  ( isLetter, isSpace )
-import qualified Data.HashMap.Lazy          as HashMap
+
+#if MIN_VERSION_aeson(2,0,0)
+import qualified Data.Aeson.KeyMap          as Map
+#else
+import qualified Data.Aeson.Parser          as JSON ( json' )
```

Comment:
i am being preserving this import although the compiler warns it is unused, maybe it was not reexported in some aeson version?

Reply:
I think that's from a bad merge between this PR and https://github.com/ennocramer/floskell/pull/70. The current status of the file shows `Data.Aeson.KeyMap` being imported twice: once as `JSON` (#70) and once as `Map` (#69). Also it looks like this PR erased the CPP in mine at the use site, so I'd expect there to be errors now for anyone trying to use aeson<2.0

Reply:
The `CPP` at the use site is not necessary, as both `Data.HashMap.Lazy` and `Data.Aeson.KeyMap` are imported as `Map` now, so the call is `Map.unionWith` either way.

I'll merge @jneira 's followup PR now but will leave the Hackage release unchanged.